cql/pom.xml

```diff
@@ -128,7 +133,7 @@
                             org.javatuples,
                             com.codahale.metrics,
                             io.micrometer.core.*,
-                            com.datastax.oss.driver.shaded.guava.*,
+                            com.datastax.oss.driver.shaded.guava.common.*,
```

Comment:
so what's with the `.common` extra?

Reply:
I saw an issue earlier for odd import of packages under "shaded.guava." (for something like `...guava.appengine.api`), and everything we need is under "shaded.guava.common.". So it is just for minimal inclusion -- seems like we should use this limited inclusion everywhere. Would make sense to be consistent.